### ISSUE TITLE
pl-string-input and pl-symbolic-input eat spaces from my labels when using math

### ISSUE THREAD

**Issue by cbm755** [PrairieLearn/PrairieLearn#4371]
Inside a pl-string-input or a pl-symbolic-input, consider these labels:
  * `label="$L$ is "`
  * `label="$y\;$ is"`
  * `label="$y$&nbsp;is"`

In the first one, something eats the space between L and `is` so I visually get `Lis`.  Why?  The second two are possible workaround.

Renderings:

![Screenshot from 2021-06-13 10-40-32](https://user-images.githubusercontent.com/818622/121817031-d098b800-cc33-11eb-87b4-51fdd017dd74.png)

Note acceptable "y is" and very poor "Lis".  (Ignore the `y is` versus `$x = $`, this is not my real quesion.)

**Comment by eliotwrobson** [comment]
Is this something unavailable? It seems like it has to do with the way bootstrap handles content rendered by mathjax.

**Comment by cbm755** [comment]
> Is this something unavailable?

Do you mean the repo linked to?  I think its private (its my course PL content repo).

There isn't any more detail there except some workarounds:

![image](https://user-images.githubusercontent.com/818622/233871133-2ba8bc8f-2401-45d8-86d4-f24539429a48.png)

**Comment by eliotwrobson** [comment]
@cbm755 oh I meant to say "is this something unavoidable?" (autocorrect, ugh). But I'm wondering if the best way to resolve this is to just to add the workaround to the docs somewhere.

**Comment by cbm755** [comment]
Yes, that makes sense...  Is it in the `mathjax` or `Bootstrap` FAQs?  I assume I looked back when I filed this but not sure I even knew what Bootstrap was back then...

**Comment by cbm755** [comment]
Relevant?

https://stackoverflow.com/questions/26111684/mathjax-display-issues-breaking-formatting-in-certain-places-only

**Comment by cbm755** [comment]
https://stackoverflow.com/questions/40149226/bootstrap-input-group-with-mathjax
